package Exception_IO.Challanges.ch1;

public class Contact {
    String name;
    String phoneNumber;
}
